drivers/ethernet/eth_adin2111_priv.h

```diff
@@ -157,7 +158,13 @@
 /* Number of buffer bytes in TxFIFO to provide frame margin upon writes */
 #define ADIN2111_TX_FIFO_BUFFER_MARGIN		4U
 
+enum adin2111_chips_id {
+	ADIN2111_MAC = 0,
+	ADIN1110_MAC,
+};
+
 struct adin2111_config {
+	enum adin2111_chips_id id;
```

Comment:
I would recommend to change the enum naming convention to reflect it is now supporting a family of chips and not just a single variant.

Reply:
Probably follows the overall naming scheme.
Could be addressed when handling https://github.com/zephyrproject-rtos/zephyr/pull/60619#discussion_r1270543164

Reply:
it seems that the linux driver also goes for the first part supported as naming convention:
https://github.com/torvalds/linux/blob/c1a515d3c0270628df8ae5f5118ba859b85464a2/drivers/net/ethernet/adi/adin1110.c#L119